File: Python/Modul2_Percobaan3_Kel46.py
angka = float(input('Masukkan angka yang anda inginkan = '))
if angka % 2 == 0:
	print('Angka yang anda masukkan adalah genap yaitu %s' %angka)
elif angka % 2 == 1:
	print('Angka yang anda masukkan adalah ganjil yaitu %s' %angka)
else:
	print('Angka yang anda masukkan bukan bilangan bulat yaitu %s' %angka)
print('Program Selesai')

File: Java/Modul2_Percobaan3_Kel46.java
import java.util.Scanner;

public class Modul2_Percobaan3_Kel46 {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Masukan Angka = ");
        int i = sc.nextInt();
        if(i == 0){
            System.out.println("Angka yang dimasukan adalah 0");
        }
        else if(i>0) {
            System.out.println("Bilangan Bulat Positif yaitu "+i);
        }
        else if(i<0){
            System.out.println("Bilangan Buat Negatif yaitu "+i);
        }
        System.out.println("Program Selesai");
    }
}